README.md

```diff
@@ -156,16 +185,16 @@ Follow the above instructions to build LLVM or acquire a suitable binary
 release. Then change directory to the Halide repository and run:
 
 ```
-% cmake -DCMAKE_BUILD_TYPE=Release -DLLVM_DIR=$LLVM_ROOT/lib/cmake/llvm -S . -B build
+% cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DLLVM_DIR=$LLVM_ROOT/lib/cmake/llvm -S . -B build
```

Comment:
Doesn't this also require acquiring ninja, would would need to be added to the instructions? I'm not sure it's a good default.

Reply:
I made this change for two reasons, let me know what you think:

1. We only use Ninja with CMake on the buildbots, so if there are issues with the Makefile backend, we won't notice them.
2. The user experience with Ninja is much nicer (parallel by default, cleaner output, etc.)

Also, it's perfectly optional and one can avoid ninja by simply dropping `-G Ninja`. I wrote a note on that below.

Reply:
I think that people copy-paste build steps without reading or understanding them, and that this one isn't going to work without first telling them to acquire ninja. Maybe the text below could more explicitly say that if you get the error <exact error string>, try again without -G Ninja